repos/iiif-storage/src/Aggregate/DereferencedManifest.php

```diff
@@ -45,7 +45,7 @@ public function mutate(ItemRequest $input)
     {
         foreach ($input->getValue('dcterms:identifier') as $field) {
             $manifestUrl = $field->getId();
-            if ($manifestUrl) {
+            if ($manifestUrl && $manifestUrl !== 'uri:internal') {
```

Comment:
This isn't ideal, should possibly use a different identifier, or a completely different field to branch from. This is to bypass the validation from the resource template / class, since it requires a URL. It's valuable to have this validation, as it avoid manifests being imported incorrectly. However, it does make it a pain to make non-external manifests.

This could be fixed by splitting the resource templates into: `External IIIF Manifest` and `Custom IIIF Manifest` or something.